### PR TITLE
Remove "gypsy" slur from documentation.

### PR DESCRIPTION
**Pull request by cowlicks**
Gypsy is considered a slur by the Romani people whom it refers to. "manouche jazz" is used in place of "gypsy jazz" and is an accepted term for the same genre of music.

### CODE REVIEW

**Comment by cowlicks** [comment]
I also found "gipsy" in a test file. This has been removed too.

**Comment by aaugustin** [comment]
This change seems acceptable both from an English and French perspective. (I'm bringing up French because we're talking of Django Reinhardt here.)

I don't feel qualified to comment on the wider Romani perspective; I think it will depend on countries and languages anyway.

For what it's worth, in French, the word "manouche" used in isolation is more pejorative than "gitan". (For example, police officers typically says "manouche" to designate informally the non-law-abiding fraction of this community.) But "manouche" seems to loses that connotation in "manouche jazz" or "jazz manouche", so I believe it's fine.

**Comment by aaugustin** [comment]
I'm wondering whether "gypsy" similarly loses its pejorative connotation in "gypsy jazz"... Now we just need to find a Romani living in an English speaking country who's familiar with the French music scene from the 30s and 40s, how hard can it be ;-)

Pinging @adrianholovaty in case he has an opinion on this.

**Comment by adrianholovaty** [comment]
My two cents: jazz manouche would be fine.

**Comment by cowlicks** [comment]
Switched to jazz manouche.

File: docs/spelling_wordlist [hunk 1]
```diff
@@ -455,6 +455,8 @@ Magee
 makemessages
 makemigrations
 Mako
+manouche
+Manouche
```
Comment (timgraham) [inline docs/spelling_wordlist]: I think the uppercased version isn't required in addition to the lower cased version.

**Comment by timgraham** [comment]
Could you please squash commits?

**Comment by cowlicks** [comment]
@timgraham done

**Comment by timgraham** [comment]
merged in 5fdbd9e36c861cb9f2cdb255a1bf693de8876f6b, thanks.

**Pull request closed without merging**